doc/maintenance.md

```diff
@@ -0,0 +1,538 @@
+## Applying the HTTP methods to the SDMX REST API for Structural Metadata maintenance
+
+The following sections describe the foreseen HTTP methods for the maintenance of SDMX Structural Metadata via the REST API.
+
+
+### CREATE
+
+Inserting or creating new SDMX Structural Metadata is detailed in this secion.
+
+#### Artefact (Structure)
+
+Submitting SDMX Artefacts in bulk, either of the same or of different types, is achieved with an HTTP `POST` method.
+Creating new Artefact(s) may be issued by:
+- `POST` one or more Maintainable Artefacts under the proper resource type, e.g. for Codelists: `/structure/codelist/`
+- `POST` one or more Maintainable Artefacts under the abstract structure resource type, e.g. `/structure/`
+
+##### Client
+
+In order to create Artefacts, the client:
+
+- MAY set the `Accept` header to indicate the preferred response format;
+- MUST set the `Content-type` header according to the format of the submitted Artefact(s);
+- MUST include in the request body, one or more Maintainable Artefacts in the SDMX format indicated in the `Content-type` header and of the SDMX type indicated in the resource, i.e.:
+  - any set of Maintainable Artefacts under resource `/structure/`
+  - a set of specific type of Maintainable Artefacts under the corresponding resource type, e.g. for Codelists: `/structure/codelist/`
+
+##### Server
+
+In response to Artefact(s) creation, the server:
+
+- MUST return `201` upon successful creation (or `207` for partial success);
+- MUST return a `SubmitStructureResponse` message with the result of the action(s), according to the `Accept` header, or the default, if the `Accept` type is not supported (currently available only in SDMX-ML format);
+- MUST set the `Content-type` according to the returned format;
+- MAY set the `Location` header to point to the created/primary resource/Artefact (only one instance is allowed);
+
+##### Response
+
+The `SubmitStructureResponse` message must be returned in any case (success, partial success, failure).
+This is defined as part of the `RegistryInterface` messages.
+The details of the message are explained in section **Structure response message**, below.
+
+Especially when different results occur on the Artefacts (e.g. partial success), the server should act as follows:
+- Return a multi-status return code (like `207`);
+- Return a `JSON` or `XML` message with the details of the result (currently available only in SDMX-ML);
+
+| Method | Exists | Versioning respected | Is Referenced | References exist/provided | Response Code |
+|---|---|---|---|---|---|
+| POST | F | T | - | T | `201` (successful) or `207` (partially successful) |
+| POST | F | T | - | F | `409` missing references |
+| POST | F | F | - | I | `409` failure to respect the versioning rules |
```

Comment:
what happens if exists is T?

Reply:
Since this section examins creating a new artefact, `exists` cannot be `true`. Otherwise, it falls under the 'update' case.